config/locales/en.yml

```diff
@@ -530,6 +530,13 @@ en:
         all: All
         limited: Limited
         title: Moderation
+      notes:
+        create: Add Internal Note
+        created_msg: Instance note successfully created!
+        destroyed_msg: Instance note successfully deleted!
+        placeholder: Information about this instance, actions taken, or anything else that will help you moderate this instance in the future.
+        title: Internal Notes
```

Comment:
I'm using "Internal Notes" as the heading to emphasize that these aren't visible to the general public.

There is also still "private_comment" vs "public_comment" on DomainBlocks, but I suspect we'll see people opt not to use private comments in the future (plus they're usually junk because of how importing domain blocks works)

Reply:
Could also use "Private Notes" maybe, or "Moderator Notes", idk.